.github/workflows/push.yaml

```diff
@@ -0,0 +1,65 @@
+name: push
+
+on:
+  push:
+    branches:
+      - '**'
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: 3.2.2
+      - run: bundle install
```

Comment:
I believe `ruby/setup-ruby` runs `bundle install` automatically?

Reply:
just checked and you're correct! it will run `bundle install` if you've set `bundler-cache: true` (which we have); i'll remove `bundle install`